.env

```diff
@@ -0,0 +1,4 @@
+DB_HOST = localhost
+DB_USER = root
+DB_PASSWORD = root
+DB_NAME = proposals_mngmt
```

Comment:
@tusharncuelogic @sandeepp-cuelogic @gauravc-cuelogic @pritig-cuelogic all above variables should be in the environment. Like this USER_ID=239482 USER_KEY=foobar node app.js

and you should access those process.env.USER_ID.